apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/http/ScanHttpEncodings.scala

```diff
@@ -489,7 +488,7 @@ object ScanHttpEncodings {
     httpApi.EventHistoryVerdict(
       updateId = verdict.updateId,
       migrationId = verdict.migrationId,
-      domainId = verdict.domainId.toString(),
+      domainId = Codec.encode(verdict.domainId),
```

Comment:
FYI @dfordivam we use this quite often in the http APIs to ensure we get correct output. We somehow missed that you were not using it